mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td

```diff
@@ -1825,4 +1825,42 @@ def LLVM_CallIntrinsicOp
   let hasVerifier = 1;
 }
 
+def LLVM_LinkerOptionsOp
+    : LLVM_Op<"linker.options"> {
```

Comment:
MLIR has a strong preference for underscores over dots. Can we change the op name to `linker_options`?